### ISSUE TITLE
Add mappings for default sort keys

### ISSUE THREAD

**Issue by matthewhanson** [stac-utils/stac-server#249]
If a collection is ingested (and therefore an Elasticsearch index exists), but no Items have been ingested, then the 
`/search` endpoint throws an error (due to an elasticsearch error)

Not entirely sure if it's due to the default sort keys of ['properties.datetime', 'id', 'collection'] but seems like it could be, since those fields or mappings do not exist. 

The problem is avoided as long as there is a single item, even if new collections are added but are empty of items.

In other words this problem only occurs when there are no items in the backend, so not really an immediate problem.

Recommend adding explicit mappings so they exist and test the /search endpoint.
